webdriver-spec.html

```diff
@@ -4105,7 +4108,46 @@
 
  <li><p><a>Scroll into view</a> the <var>element</var>.
 
- <li><p class=issue>Interact with the element.
+ <li><p>Run the <a>focusing steps</a> for the <a>element</a>.
+
+ <li><p>Set the cursor position to after the last character in the element
+
+ <li><p>Let <var>character array</var> be the result of <a>getting a property</a>
+    <code>text</code> from the <var>parameters</var> argument.
+
+ <li><p>If <var>element</var> is a <a><code>input</code></a> whose
+   <a><code>type</code> attribute</a> is <a>File</a>:
+   <ol>
+     <li><p>Let <var>text</var> be the result of
+      joining together each element in <var>character array</var> to a string.
+
+     <li><p>Set the <a><code>value</code> Attribute</a> to <var>text</var>.
```

Comment:
This isn’t what current implementations are doing.  We need to take into account that the HTML specification doesn’t allow you to set the value of `<input type=file>`.

This necessitates implementation-specific steps and we ought to call out this out.

For example the Gecko implementation does the following in privileged chrome space:

```
let val = "file.txt";
let fs = Array.prototype.slice.call(fileInputElement.files);
let file;
try {
  file = new File(val);
} catch (e) {
  throw new InvalidArgumentError("File not found: " + val);
}
fs.push(file);
el.mozSetFileArray(fs);
```

So:

* Add test for file existence
* Allow multiple files to be appended to existing array of files on element